addons/binding/org.openhab.binding.rfxcom.test/src/test/java/org/openhab/binding/rfxcom/internal/messages/RFXComChimeMessageTest.java

```diff
@@ -8,21 +8,36 @@
  */
 package org.openhab.binding.rfxcom.internal.messages;
 
+import static org.junit.Assert.assertEquals;
+import static org.openhab.binding.rfxcom.internal.messages.RFXComChimeMessage.SubType;
+
+import javax.xml.bind.DatatypeConverter;
+
 import org.junit.Test;
-import org.openhab.binding.rfxcom.internal.exceptions.RFXComMessageNotImplementedException;
-import org.openhab.binding.rfxcom.internal.messages.RFXComBaseMessage.PacketType;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+
 
 /**
  * Test for RFXCom-binding
  *
+ * @author Mike Jagdis
  * @author Martin van Wingerden
  * @since 1.9.0
  */
 public class RFXComChimeMessageTest {
 
-    @Test(expected = RFXComMessageNotImplementedException.class)
-    public void checkNotImplemented() throws Exception {
-        // TODO Note that this message is supported in the 1.9 binding
-        RFXComMessageFactory.createMessage(PacketType.CHIME);
+    @Test
+    public void testSomeMessages() throws RFXComException {
+        String hexMessage = "0716020900A1F350";
```

Comment:
You could consider adding a test based on the input given in the issue.

Then we know it for sure.